CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 ## Unreleased
 
 ### Features
-...
+- Add Catalan and Polish support to `detectLanguage`.
+- Add utility function `run_is_already_computed` to `Runner` to check if a run with the given metadata has already been computed.
```

Comment:
you don't mention the `retrieve_previous_runs`. I would also mention the changes to the notebook so people know how to use it